llvm/test/TableGen/GlobalISelEmitter-implicit-defs.td

```diff
@@ -0,0 +1,13 @@
+// RUN: llvm-tblgen -gen-global-isel -warn-on-skipped-patterns -I %p/../../include -I %p/Common %s -o /dev/null 2>&1 < %s | FileCheck %s
+
+include "llvm/Target/Target.td"
+include "GlobalISelEmitterCommon.td"
+
+// CHECK-NOT: Skipped pattern: Src pattern result has 1 def(s) without the HasNoUse predicate set to true but Dst MI has no def
```

Comment:
CHECK-NOT should be used sparingly, and with a very broad check. This is a very specific error message. If you want negative checks, maybe better to use --implicit-check-not=pattern

Reply:
Ah, I see. Because if that error message changes a tiny bit, it'll succeed, even if there's a regression. I'll do `--implicit-check-not="Skipped pattern"` then